jcl/src/jdk.jcmd/share/classes/openj9/tools/attach/diagnostics/tools/Jps.java

```diff
@@ -33,12 +33,27 @@
  * A tool for listing Java processes and their information.
  *
  */
+@SuppressWarnings("nls")
 public class Jps {
 
+	private static final String DESCRIPTION =
+			"jps: Print a list of Java processes and information about them%n%n";
+
+	private static final String HELP_TEXT =
+			"usage: jps [-h] [-J] [-l] [-q] [-m] [-v]%n" +
+			"usage: jps [-h] [-Jlqmv]%n%n" +
```

Comment:
-J needs to be a separate argument. It's actually `-J<vm argument>` such as `-J-Xmx512m`.

Reply:
There are duplicated `usage:` here, in addition, the usage format should be consistent with other tools such as `Jmap`, `Jstack`, and `Jstat` in same package.

Reply:
@JasonFengJ9, sure, will remove the duplicate. And by the "consistent format" do you mean to have support of composed arguments for every tool?

Reply:
@alexey-anufriev I meant the usage format like `Jmap` as following:
```
	private static String HELPTEXT = "jmap: obtain heap information about a Java process%n"
			+ " Usage:%n"
			+ "    jmap <option>* <vmid>%n"
```
in which `Usage:%n` is in a separated line.

> All the other tools in the same package can be updated similarly.

I think this is about the support of composed arguments for other tools though not necessary in this PR.

Reply:
@JasonFengJ9, ah, so you mean the help output. For sure I can adapt that.

Reply:
Improved